### DISCUSSION TITLE
Dose 3 support?

### DISCUSSION THREAD

**Discussion by airowe-ra** [smart-on-fhir/health-cards#173] [Ideas] [answered]
Are there any modifications we need to make for Dose 3 support in the JWS payload? There are currently only resource2 and resource3 nodes to represent vaccinations.

**Comment by jmandel** [comment, marked as answer]
We don't have any specific advice on this. It's one of the advantages of our [advice not to encode intended protocols](https://build.fhir.org/ig/HL7/fhir-shc-vaccination-ig/vaccination.html#why-protocolapplied-is-not-allowed) into the immunization resources -- you never need to say "dose 3 of 2" or represent a "whoops, change of plans" later on.

So: the expectation is that issuers will produce a single Health Card incorporating all COVID-19 immunization doses they're aware of.

**Comment by arztnh** [comment]
Remember, the current situation in t he US is for "Additional Doses"
    to be administered to the immunocompromised, and "Booster Doses" to
    likely be delivered soon to everyone (terminology is important to
    some). But CDC has not defined (or redefined) what
    "fully-vaccinated" means yet, or will mean if the booster doses go
    through as expected.
    
    Noam
    
    On 8/25/2021 11:27 AM, Josh Mandel
      wrote:
    
    
      
      We don't have any specific advice on this. It's one of the
        advantages of our advice not to encode
          intended protocols into the immunization resources -- you
        never need to say "dose 3 of 2" or represent a "whoops, change
        of plans" later on.
      So: the expectation is that issuers will produce a single
        Health Card incorporating all COVID-19 immunization doses
        they're aware of.
      —
        You are receiving this because you are subscribed to this
        thread.
        Reply to this email directly, view it on GitHub, or unsubscribe.
        Triage notifications on the go with GitHub Mobile for iOS or Android.
      [
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/smart-on-fhir/health-cards/discussions/173#discussioncomment-1234793",
"url": "https://github.com/smart-on-fhir/health-cards/discussions/173#discussioncomment-1234793",
"name": "View Discussion"
},
"description": "View this Discussion on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]
